code/modules/modular_computers/file_system/programs/generic/ntnrc_client.dm

```diff
@@ -156,6 +166,28 @@
 			channel.password = ""
 		else
 			channel.password = newpassword
+	if(href_list["PRG_directmessage"])
+		. = TRUE
+		var/clients = list()
+		var/names = list()
+		for(var/cl in ntnet_global.chat_clients)
```

Comment:
You should be able to message anyone with an active modular computer, not just active on the IRC channels.

Reply:
You can message anyone who are currently online. but not anyone having client program. You can minimize client and still get messages.